src/styl/_variables/_color.styl

```
@@ -3,6 +3,87 @@
 // Every design’s colors are declined as CSS custom properties.
 //
 // Styleguide: CSSvariables.color
+
+// Colors
+//
+// Every colors in this Colettes are variables.
+// If an overwrite of the color is needed, checkout <a href="http://developers.20minutes.fr/colette/section-settings.html#kssref-settings-colors" style="text-decoration: underline">the stylus variables on the settings</a>
```

```suggestion
// If an overwrite of the color is needed, checkout [the stylus variables on the settings](./section-settings.html#kssref-settings-colors).
```
The underline style on link in description should be into colette-kss-builder css